lang/en.js

```diff
@@ -25,6 +25,9 @@ export default {
 	"components.filter.searchResults": "{number, plural, =0 {No search results} one {{number} search result} other {{number} search results}}",
 	"components.filter.selectedFirstListLabel": "{headerText}. Selected filters appear first.",
 	"components.filter.singleDimensionDescription": "Filter by: {filterName}",
+	"components.filter-dimension-set-date-text-value.textHours": "{num, plural, one {Last hour} other {Last {num} hours}}",
+	"components.filter-dimension-set-date-text-value.textDays": "{num, plural, =0 {Today} one {Last {num} days} other {Last {num} days}}",
```

Comment:
The LocalizeMixin sandbox auto-adds the `one` case for several languages (I just had `=0` and `other`). We don't currently have a case for one, though it would probably be "yesterday" if there was anything. Not sure if I should leave as-is or change.